akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphMatValueSpec.scala

```diff
@@ -227,5 +227,12 @@ class GraphMatValueSpec extends StreamSpec {
       matValue should ===(NotUsed)
 
     }
+
+    "not ignore materialized value of indentity flow which is optimized away" in {
+      implicit val mat = ActorMaterializer(ActorMaterializerSettings(system).withAutoFusing(false))
+      val (m1, m2) = Source.single(1).viaMat(Flow[Int])(Keep.both).to(Sink.ignore).run()
```

Comment:
`val (NotUsed, NotUsed) = Source.single(1).viaMat(Flow[Int])(Keep.both).to(Sink.ignore).run()`

No assertions needed :)